mlrun/__main__.py

```diff
@@ -950,7 +950,9 @@ def project(
         proj.spec.params["commit_id"] = commit
     if secrets:
         secrets = line2keylist(secrets, "kind", "source")
-        proj._secrets = SecretsStore.from_list(secrets)
+        secret_store = SecretsStore.from_list(secrets)
+        proj._secrets = secret_store
+        proj.set_secrets(secret_store._secrets)
```

Comment:
If you are running `proj.set_secrets(secret_store._secrets)`, is `proj._secrets = secret_store` still necessary?

Reply:
I did not want to change the original flow in order to not break anything just add the project.set_secrtes.
But maybe we can remove it